services/gemini-1/docker-compose-extra.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+
+volumes:
+  archival_node_data: {}
+
+services:
+  archival-node:
+    image: ghcr.io/subspace/node:${NODE_SNAPSHOT_TAG}
+    volumes:
+      - archival_node_data:/var/subspace:rw
+    restart: unless-stopped
+    ports:
+      - "30333:30333"
+    command: [
+      "--chain", "gemini-1",
+      "--base-path", "/var/subspace",
+      "--execution", "wasm",
+      "--pruning", "archive",
+      "--pool-kbytes", "51200",
+      "--listen-addr", "/ip4/0.0.0.0/tcp/30333",
+      "--rpc-cors", "all",
+      "--rpc-external",
+      "--ws-external",
+      "--in-peers", "500",
+      "--out-peers", "250",
+      "--in-peers-light", "500",
+      "--ws-max-connections", "10000",
```

Comment:
Is there a patch of a base linux kernel? AFAIK it supports no more than 1000 per service in default settings

Reply:
We had issues with 1000 and no issues once we raised it to 10k so my impression is that is should work, but maybe I'm wrong about it :man_shrugging:

Reply:
```root@gemini-1b-node-extra-9-sfo3:~# sysctl -a | grep soma
net.core.somaxconn = 4096```

4k ws allowed

Reply:
Then yes, we need to tune this. I'm sometimes wondering why those limits are so low on Linux...

Reply:
I dont think we need to a kernel patch. I think updating systconf to required connections should be fine in this case